### PR TITLE
Update DistortableImage.Edit.js

### PR DESCRIPTION
**Pull request by leilayesufu**
Fixes #1086  (<=== Add issue number here)

Make sure these boxes are checked before your pull request (PR) is ready to be reviewed and merged. Thanks!

* [ ] PR is descriptively titled 📑 and links the original issue above 🔗
* [ ] tests pass -- look for a green checkbox ✔️ a few minutes after opening your PR -- or run tests locally with `grunt test`
* [ ] code is in uniquely-named feature branch and has no merge conflicts 📁
* [ ] screenshots/GIFs are attached 📎 in case of UI updates
* [ ] @mention the original creator of the issue in a comment below for help or for a review

> We're happy to help you get this ready -- don't be afraid to ask for help, and **don't be discouraged** if your tests fail at first!

If tests do fail, click on the red `X` to learn why by reading the logs.

Please be sure you've reviewed our contribution guidelines at https://publiclab.org/contributing-to-public-lab-software 

Thanks!

### CODE REVIEW

**Comment by leilayesufu** [comment]
Hello @TildaDares , thank you for assigning me this issue
kindly review my pr

**Comment by leilayesufu** [comment]
@jywarren kindly review my pr

File: src/edit/DistortableImage.Edit.js [hunk 1]
```diff
@@ -397,25 +397,25 @@ L.DistortableImage.Edit = L.Handler.extend({
...
-      var width = downloadable.width;
-      var nw = map.latLngToLayerPoint(overlay.getCorner(0));
-      var ne = map.latLngToLayerPoint(overlay.getCorner(1));
-      var sw = map.latLngToLayerPoint(overlay.getCorner(2));
-      var se = map.latLngToLayerPoint(overlay.getCorner(3));
+      let height = downloadable.height;
+      let width = downloadable.width;
+      let nw = map.latLngToLayerPoint(overlay.getCorner(0));
+      let ne = map.latLngToLayerPoint(overlay.getCorner(1));
+       sw = map.latLngToLayerPoint(overlay.getCorner(2));
```
Comment (TildaDares) [inline src/edit/DistortableImage.Edit.js:417]:
```suggestion
      let sw = map.latLngToLayerPoint(overlay.getCorner(2));
```

**Comment by leilayesufu** [comment]
I’ve updated the changes, thank you
@TildaDares

**Review by TildaDares — APPROVED** [review]
Amazing work @leilayesufu. Thank you!!

**Pull request merged**